packages/react-data-grid/src/common/editors/CheckboxEditor.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+interface Column<T> {
```

Comment:
maybe we should consider using some of the defs from my PR?
https://github.com/adazzle/react-data-grid/pull/1466/files#diff-bee94f77d0ed359d978babfd95123920R309

Reply:
I'd like to define it as needed/incrementally, to make sure we get it 100% right, and not make mistakes ahead of time.